internal-enrichment/reversinglabs-spectra-analyze/src/settings.py

```diff
@@ -0,0 +1,42 @@
+from typing import Literal
+
+from connectors_sdk import (
+    BaseConfigModel,
+    BaseConnectorSettings,
+    BaseInternalEnrichmentConnectorConfig,
+)
+from pydantic import Field
+
+
+class ReversinglabsSpectraAnalyzeConfig(BaseConfigModel):
+    url: str = Field(description="API base URL", validation_alias="spectra_analyze_url")
+    token: str = Field(
+        description="API token", validation_alias="spectra_analyze_token"
```

Comment:
```suggestion
    url: str = Field(description="API base URL", validation_alias="url")
    token: str = Field(
        description="API token", validation_alias="token"
```
to be consistent with the config.yml